Dashboard/app/js/lib/controllers/survey-controllers.js

```diff
@@ -697,7 +706,12 @@ FLOW.surveyControl = Ember.ArrayController.create(observe({
         return r;
     }, Object.create(null));
 
-    return questionGroupsNoName.reduce((c, o) => { if(!c[o.get('keyId')]) { c[o.get('keyId')] = [];} return c;} , data);
+    const groups = questionGroupsNoName.reduce((c, o) => { if(!c[o.get('keyId')]) { c[o.get('keyId')] = [];} return c;} , data);
```

Comment:
I am just curious what do c and o stand for?

Reply:
Great question! I've renamed the variables to make them more readable.